lib/consumer.js

```diff
@@ -69,8 +89,7 @@ class BigIotConsumer extends Client {
           offeringId,
         },
       },
-    })
-      .then(result => result.data.subscribeConsumerToOffering);
+    }).then(result => result.data.subscribeConsumerToOffering);
```

Comment:
There are alot of irrelevant code formatting changes. Can you please remove them?